packages/storefront/src/pages/license.md

```diff
@@ -56,3 +29,14 @@ on the exact product and version you choose.
 This software may or may not contain the following specific Open Source Software:
 
 <p-link-pure icon="download" href="assets/compulsory-statement.txt" download>Download Compulsory Statement</p-link-pure>
```

Comment:
![image](https://github.com/porsche-design-system/porsche-design-system/assets/109523972/1b54ae67-0a8b-473f-9d36-a5728c3251f2)
Auto dark mode does not work here

Reply:
I added it to https://github.com/porsche-design-system/porsche-design-system/issues/2779